CMakeLists.txt

```diff
@@ -25,8 +25,15 @@ endif ()
 set(CMAKE_C_COMPILER_NAMES clang-7 cc)
 set(CMAKE_CXX_COMPILER_NAMES clang++-7 c++)
 
-project("Open Enclave SDK" LANGUAGES C CXX ${OE_ASM})
+project("Open Enclave SDK" LANGUAGES C CXX ${OE_ASM}
+  HOMEPAGE_URL "https://github.com/openenclave/openenclave")
 set(PROJECT_VERSION ${OE_VERSION})
+set(OE_SCRIPTSDIR "${PROJECT_SOURCE_DIR}/scripts")
```

Comment:
Since maybe_build_using_clangw is used during both the normal build of the SDK and by a consumer of the SDK, it will need to know how to find the differently-pathed clangw/llvm-arw scripts. I thought that the easiest way to abstract this was to simply make it be a variable that can be defined differently depending on whether it comes from the build for the SDK or from the SDK package itself.